#WorldInfo.msg
float64[3] world_dim
int32[3] grid_dim